### PR TITLE
Document tplink_omada update entities

### PR DESCRIPTION
**Pull request by MarkGodwin**
## Proposed change

Adding Update entities to the TP-Link Omada devices. This doesn't require a lot of documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [89562](https://github.com/home-assistant/core/pull/89562)
- Link to parent pull request in the Brands repository: N/A
- This PR fixes or closes issue: fixes # N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tplink_omada.markdown [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: TP-Link Omada SDN Controller
 description: Instructions on integrating TP-Link Omada SDN networking devices to Home Assistant.
 ha_category:
   - Hub
-ha_release: 2023.3
+ha_release: 2023.4
```
Comment (frenck) [inline source/_integrations/tplink_omada.markdown]: This should not be updated. It indicates the release the integration was introduced, which will not change :)

```suggestion
ha_release: 2023.3
```

File: source/_integrations/tplink_omada.markdown [hunk 2]
```diff
@@ -20,11 +20,12 @@ The integration provides basic configuration and status of Omada devices control
 
 {% include integrations/config_flow.md %}
 
-- Host: Enter the URL of the Omada management interface.
+- Host: Enter the URL, DNS Name or IP Address of the TP-Link Omada Controller. HTTPS will be used be default, if not specified.
```
Comment (frenck) [inline source/_integrations/tplink_omada.markdown]: Unrelated change, this should not be in this PR.

File: source/_integrations/tplink_omada.markdown [hunk 3]
```diff
@@ -42,14 +43,15 @@ Controller versions 5.0.0 and later are supported.
 ### Network Switches
 
 - Support for enabling/disabling Power over Ethernet on a per-port basis.
+- Firmware Update notification and installation.
 
 ### Access Points
 
-- Not currently supported.
+- Firmware Update notification and installation.
```
Comment (frenck) [inline source/_integrations/tplink_omada.markdown]: It is not a notification? It is an update entity, right?

Notifications are something one could create with those, but that is not something we provide?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tplink_omada.markdown (hunk 2 above)
Comment (MarkGodwin) [reply source/_integrations/tplink_omada.markdown]: Fair point. I've reverted.

File: source/_integrations/tplink_omada.markdown (hunk 1 above)
Comment (MarkGodwin) [reply source/_integrations/tplink_omada.markdown]: Yeah, that makes a lot more sense :)

**Review by frenck — APPROVED** [review]
Thanks, @MarkGodwin 👍 

../Frenck
